src/commands/bootcamp/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { max } from 'lodash';
+
+export const toTitleCase = (str: string): string => {
+  return str.replace(/-/g, ' ').replace(/\w\S*/g, function (txt: string) {
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+  });
+};
+
+export const lowestInt = (nums: number[]): number => {
```

Comment:
is this the smallest int that's missing from `nums`? the function name could be better and documentation is needed